src/core/lib/channel/channelz.h

```diff
@@ -58,12 +58,8 @@ namespace channelz {
 grpc_arg MakeParentUuidArg(intptr_t parent_uuid);
 intptr_t GetParentUuidFromArgs(const grpc_channel_args& args);
 
-// TODO(ncteisen), this only contains the uuids of the children for now,
```

Comment:
We should probably retain this TODO somewhere.  API-wise, we'll probably want to do this by adding a string parameter to all of the `AddChild*()` methods in both `ChannelNode` and `ServerNode`.